hatch_build.py

```diff
@@ -0,0 +1,53 @@
+from functools import partial
+from io import StringIO
+from logging import shutdown
+from pathlib import Path
+import os
+import shlex
+from subprocess import call
+import sys
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+
+class LarkStandAloneBuildHook(BuildHookInterface):
+    PLUGIN_NAME = "lark_standalone"
+
+    def initialize(self, version, build_data):
+        L = get_logger()
+        WIN = os.name == "nt"
+        L.info("converting json grammar to python")
+        python_parser = Path("src/importnb/_json_parser.py")
```

Comment:
Build tools often change location:

```suggestion
        python_parser = Path(self.root, "src/importnb/_json_parser.py")
```